backend/src/User/user.service.ts

```diff
@@ -1,33 +1,64 @@
 import UserModel from './user.model';
+import { config } from '../../config';
+
+const bcrypt = require('bcrypt');
+const jwt = require('jsonwebtoken');
```

Comment:
We should use the typescript import instead of require

Reply:
same here